client/src/app.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var app = angular.module('sunfishApp', ['ngRoute']);
+'$compileProvider',
+    app.config(['$routeProvider', '$compileProvider', function($routeProvider, $compileProvider) {
+        $routeProvider
+        .when('/', {
+            templateUrl: 'views/main/main.html',
+        })
+        .when('/browse/', {
+            controller: "BrowseCtrl",
+            templateUrl: 'views/browse/browse.html',
+        })
+        .when('/search/', {
+            controller: "SearchCtrl",
+            templateUrl: 'views/search/search.html',
+        })
+        .when('/upload/', {
+            controller: "UploadCtrl",
+            templateUrl: 'views/upload/upload.html',
+        })
+        .when('/siafile/:siafileId', {
+            controller: "SiafileCtrl",
+            templateUrl: 'views/siafile/siafile.html',
+        })
+        .otherwise({redirectTo: '/'});
```

Comment:
I'd go with the folder with several html files myself. Also 20's an exaggeration and you know it, whore :fu: